app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    com_spl = command.split()
+    cmd, file_1, path_file = com_spl
```

Comment:
Unpacking should be done after length check

And use better names for variables